storage/innobase/handler/ha_innodb.cc

```diff
@@ -1612,7 +1612,17 @@ innobase_get_cset_width(
 			}
 		} else {
 
-			ut_a(cset == 0);
+			if (cset != 0) {
+				/* Fix bug#1533722:
+				Since we don't do anything fancy with the data
+				itself, it is safe to ignore missing collation.
+				(special case for collation that isn't supported
+				in vanilla MySQL) */
+				fprintf(stderr,
```

Comment:
It is better to use `ib::warn` instead since we are inside of InnoDB code

Reply:
Unfortunately, `ib::warn` is not available in the 2.3.

Reply:
But there is `ib_logf`!